meshes/spritesheet-mesh.js

```diff
@@ -293,80 +294,87 @@ export class SpritesheetMesh extends ChunkedBatchedMesh {
     this.allocatedChunks = new Map();
   }
   addChunk(chunk, chunkResult) {
-    const instances = chunkResult;
-
-    if (chunk.lod >= this.lodCutoff && instances.length > 0) {
-      const _renderLitterSpriteGeometry = (drawCall, instances) => {
-        const pTexture = drawCall.getTexture('p');
-        const pOffset = drawCall.getTextureOffset('p');
-        const offsetTexture = drawCall.getTexture('offset');
-        const offsetOffset = drawCall.getTextureOffset('offset');
-        const itemIndexTexture = drawCall.getTexture('itemIndex');
-        const itemIndexOffset = drawCall.getTextureOffset('itemIndex');
-
-        let index = 0;
-        for (let i = 0; i < instances.length; i++) {
-          const instance = instances[i];
-          const {instanceId, ps, qs} = instance;
-
-          for (let j = 0; j < ps.length; j += 3) {
-            const indexOffset = index * 4;
-            
-            // geometry
-            const px = ps[index * 3];
-            const py = ps[index * 3 + 1];
-            const pz = ps[index * 3 + 2];
-            pTexture.image.data[pOffset + indexOffset] = px;
-            pTexture.image.data[pOffset + indexOffset + 1] = py;
-            pTexture.image.data[pOffset + indexOffset + 2] = pz;
-
-            offsetTexture.image.data[offsetOffset + indexOffset] = this.offsets[instanceId * 4];
-            offsetTexture.image.data[offsetOffset + indexOffset + 1] = this.offsets[instanceId * 4 + 1];
-            offsetTexture.image.data[offsetOffset + indexOffset + 2] = this.offsets[instanceId * 4 + 2];
-            offsetTexture.image.data[offsetOffset + indexOffset + 3] = this.offsets[instanceId * 4 + 3];
-
-            itemIndexTexture.image.data[itemIndexOffset + indexOffset] = instanceId;
-        
-            index++;
+    if (chunkResult) {

```

Comment:
I had to add this check in the building gen stuff too- do we know why the chunkResult is sometimes null? Is that expected behavior, or is it something we need to look at fixing?

Reply:
That is expected behavior, simply we set the `chunkResult` value to null if the buffer address is `null` (zero)
For instance : https://github.com/upstreet-labs/app/blob/master/pg-worker.js#L195

Reply:
Ah, got it- thanks for explaining.